client/ayon_core/pipeline/create/creator_plugins.py

```diff
@@ -297,23 +285,40 @@ def register_callbacks(self):
         Default implementation does nothing. It can be overridden to register
         callbacks for creator.
         """
-        pass
 
     @property
     def identifier(self):
         """Identifier of creator (must be unique).
 
         Default implementation returns plugin's product type.
-        """
 
-        return self.product_type
+        """
+        identifier = self.product_type
+        if is_product_base_type_supported():
+            identifier = self.product_base_type
+            if self.product_type:
+                identifier = f"{identifier}.{self.product_type}"
```

Comment:
This affects how instances of the create plugin are discovered. We have to keep it as it is without using `.`.
```suggestion
        identifier = self.product_base_type
        if not identifier:
            identifier = self.product_type
        return identifier
```